### DISCUSSION TITLE
Account can't be created

### DISCUSSION THREAD

**Discussion by JustToTest-MyStupidCode** [tuyafeng/Via#1038] [Q&A]
Whenever i try to create an account, it says either service unavailable or connection timedout, tried both global and chinese with different vpn locations.....need a solution please

**Comment by jkam007** [comment]
Have your tried to update to the latest version and implement again the procedure?

**Comment by JustToTest-MyStupidCode** [comment]
i already created an account, but couldn't login

**Reply by jkam007** [reply]
Global or Chinese sync server?

**Comment by JustToTest-MyStupidCode** [comment]
Global server gives "Service Unavailable" error, while chinese server gives
"timed out" error message.

**Reply by jkam007** [reply]
And you're using the version 5.3.0? So I think you should switch to webdav sync for convenience.

**Comment by JustToTest-MyStupidCode** [comment]
yes, v5.3.0....
Well, im using SwiftBackup to save the browser data.....Thanks
